src/common/io/tls/client.c

```diff
@@ -288,48 +287,64 @@ tlsClientOpen(THIS_VOID)
         if (verifyResult != X509_V_OK)                                                                              // {vm_covered}
         {
             THROW_FMT(                                                                                              // {vm_covered}
-                CryptoError, "unable to verify certificate presented by '%s:%u': [%ld] %s",                         // {vm_covered}
-                strZ(sckClientHost(this->socketClient)), sckClientPort(this->socketClient), verifyResult,           // {vm_covered}
-                X509_verify_cert_error_string(verifyResult));                                                       // {vm_covered}
+                CryptoError, "unable to verify certificate presented by '%s': [%ld] %s",                            // {vm_covered}
+                strZ(ioClientName(this->ioClient)), verifyResult, X509_verify_cert_error_string(verifyResult));     // {vm_covered}
```

Comment:
This went from sckClientHost to ioClientName...  but those are two different things, aren't they?  hostname vs. "name" (which seems to include the port...?)

Reply:
It's because we don't know what ioClient() implements -- does it even have a hostname? The output of this error is the same currently because SocketClient combines the home and port as it was before. But in the future this might not be the case.

Reply:
Yeah, makes sense.